src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/TierCalculationServiceTest.kt

```diff
@@ -162,7 +165,7 @@ internal class TierCalculationServiceTest {
       val tier = service.calculateTierForCrn(crn)
       standardVerify()
 
-      assertThat(tier.data.protect.points).isEqualTo(20)
```

Comment:
Every time I change anything, I seem to have to spend 20 minutes in this test despite none of the functionality changing